### ISSUE TITLE
Bring Traction forward to current state of VR2Gather.

### ISSUE THREAD

**Issue by cwidisgroup** [cwi-dis/VR2Gather#96]
Subject says all.

**Comment by cwidisgroup** [comment]
To do:
- [ ] Fix interaction so user can switch between direct and ray-based interaction
- [ ] Obejcts fly away when grabbed
- [ ] Fix second photo machine
- [ ] Test that buttons work with multiple users
- [ ] Test that grabbing works with multile users
- [ ] Test that photos work with multiple users
- [ ] Fix mailbox

**Comment by cwidisgroup** [comment]
- [ ] Point cloud is a bit too far forward.
- [ ] There is a way to start flying, @suziosaurus may remember how to. This is probably not strictly traction-related.
- [ ]

**Comment by jackjansen** [comment]
The photos had a problem that we may see again.

The Photo Booth had to be converted to be a subclass of `NetworkInstantiator` and this turned out to be non-trivial. The reason was the logic flow:
- One participant presses the "take photo button"
- All participants see the countdown
- The picture is taken (this should be done coordinated, at the master, with the UUID transmitted to all other participants so they're all the same photo)
- The photo transitions to its place in from of the users

In the end what we did was have the button be a networked button (so all participants see the countdown) and use the `Trigger()` of `NetworkInstantiator` to coordinate the creation. Then in the actual creation method we also start the final transition.

There was also an issue with the UUID not being saved until we also changed something else in the prefab instance. It may be that the auto-generation of UUIDs doesn't mark the GameObject as being dirty or something, unsure.
